src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -32,7 +35,7 @@
 		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.2.6" Condition="'$(Configuration)'=='Debug'" />
```

Comment:
Should the version for Uno.WinUI and Uno.WinUI.RemoteControl and Uno.UI.Adapter.Microsoft.Extensions.Logging be pushed to 4.5.9 to be consistent with Uno.WinUI.Lottie

Reply:
It's automatically changed by the CI to be aligned with every other packages.